calculator/index.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Why did you make this change?

Reply:
I made this line to make the code simple.